### PR TITLE
Added functionality to remove EIP addresses through the shredder

### PR DESCRIPTION
**Pull request by ArielLima**
Ticket: https://issues.redhat.com/browse/OSD-5585
Since EIPs have a hard limit of 5 per account this PR releases EIPs on failed accounts so that when reused EIPs will also reset to 0

### CODE REVIEW

**Comment by fahlmant** [comment]
Generally looks good. Please add tests

File: hack/app_sre_build_deploy.sh [hunk 1]
```diff
@@ -7,19 +7,20 @@ set -exv
 CURRENT_DIR=$(dirname "$0")
 
 BASE_IMG="aws-account-shredder"
-QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
-IMG="${BASE_IMG}:latest"
+QUAY_IMAGE="quay.io/alima/${BASE_IMG}"
```
Comment (fahlmant) [inline hack/app_sre_build_deploy.sh:10]: This should be reverted.

File: hack/app_sre_build_deploy.sh [hunk 2]
```diff
@@ -7,19 +7,20 @@ set -exv
...
 
 # build the image
 BUILD_CMD="docker build" IMG="$IMG" make docker-build
 
+docker push ${IMG}
 # push the image
-skopeo copy --dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-    "docker-daemon:${IMG}" \
-    "docker://${QUAY_IMAGE}:latest"
+# skopeo copy --dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
```
Comment (fahlmant) [inline hack/app_sre_build_deploy.sh:20]: Why was this all commented out?

**Review by fahlmant — CHANGES_REQUESTED** [review]
I think there's an unintentional commit here. Still needs tests as well

File: hack/app_sre_build_deploy.sh (hunk 1 above)
Comment (ArielLima) [reply hack/app_sre_build_deploy.sh:10]: All changes to hack need to be reverted, thank you for pointing it out

File: go.mod [hunk 3]
```diff
@@ -3,7 +3,7 @@ module github.com/openshift/aws-account-shredder
 go 1.13
 
 replace (
-	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
+	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30
```
Comment (ArielLima) [inline go.mod:6]: I couldn't get things running without this. Is it an issue on my end alone? If so I will remove the change

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**